ramp-database/rampdb/model/problem.py

```diff
@@ -51,54 +73,82 @@ def __init__(self, name):
         self.workflow_object
 
     def __repr__(self):
-        return 'Problem({})\n{}'.format(
-            encode_string(self.name), self.workflow)
+        return 'Problem({})\n{}'.format(encode_string(self.name),
+                                        self.workflow)
 
     def reset(self):
         self.workflow = Workflow.query.filter_by(
             name=type(self.module.workflow).__name__).one()
 
     @property
     def module(self):
+        """module: Get the problem module."""
         return import_module_from_source(
             os.path.join(RAMP_KITS_PATH, self.name, 'problem.py'),
             'problem'
         )
 
     @property
     def title(self):
+        """str: The title of the problem."""
         return self.module.problem_title
 
     @property
     def Predictions(self):
+        """:class:`rampwf.prediction_types`: The predictions used for the
+        problem."""
         return self.module.Predictions
 
     def get_train_data(self):
+        """ndarray: The training data."""
         path = os.path.join(RAMP_DATA_PATH, self.name)
         return self.module.get_train_data(path=path)
 
     def get_test_data(self):
+        """ndarray: The testing data."""
         path = os.path.join(RAMP_DATA_PATH, self.name)
         return self.module.get_test_data(path=path)
 
     def ground_truths_train(self):
+        """ndarray: The true labels for the training."""
```

Comment:
The return value here is not a ndarray, but a Predictions instance (and the same fot the two below)